client/boot/project/wordpress-com.js

```diff
@@ -237,18 +107,16 @@ function reduxStoreReady( reduxStore ) {
 
 	// If `?sb` or `?sp` are present on the path set the focus of layout
 	// This can be removed when the legacy version is retired.
-	page( '*', function( context, next ) {
+	page( '*', function( context, next ) { // FACTOR INTO ONE MIDDLEWARE
```

Comment:
That's my comment from below, about doing a single handler for all wildcard calls.